doc/contributing/releases.md

```diff
@@ -448,6 +448,23 @@ testing is required. Use this opportunity particularly to test macOS and Windows
 installers if there are any concerns. Click through to the individual workers
 for a run to find the artifacts.
 
+Release build list
+
+| name                                       | OS            |
+| ------------------------------------------ | ------------- |
+| rhel8-release-sources                      | ?             |
+| osx1015-release-tar                        | OSX 10.15     |
+| vs2019-x64                                 | Windows x64   |
+| cross-compiler-rhel8-arm7-gcc-8-glibc-2.28 | ?             |
+| rhel8-s390x-release                        | ?             |
+| vs2019-x86                                 | Windows x32   |
```

Comment:
```suggestion
| vs2019-x86                                 | Windows x86   |
```
FYI x32 is something else: https://stackoverflow.com/questions/7635013/difference-between-x86-x32-and-x64-architectures